app/views/core/CreateAccountModal/CreateAccountModal.coffee

```diff
@@ -114,6 +115,8 @@ module.exports = class CreateAccountModal extends ModalView
           store.commit('modal/updateSso', _.pick(@signupState.attributes, 'ssoUsed', 'ssoAttrs'))
           store.commit('modal/updateSignupForm', @signupState.get('signupForm'))
           store.commit('modal/updateTrialRequestProperties', _.pick(@signupState.get('signupForm'), 'firstName', 'lastName'))
+          if storage.load('referredBySunburst')
+            store.commit('modal/updateTrialRequestProperties', {'referredBy': 'sunburst'})
```

Comment:
The store should know about this property [here](https://github.com/codecombat/codecombat/blob/master/app/views/core/CreateAccountModal/teacher/TeacherSignupStoreModule.coffee#L21).